platform/linux-generic/odp_schedule.c

```diff
@@ -646,7 +646,7 @@ static inline void ordered_stash_release(void)
 		buf_hdr = sched_local.ordered.stash[i].buf_hdr;
 		num = sched_local.ordered.stash[i].num;
 
-		queue_fn->queue_enq_multi(queue, buf_hdr, num);
+		queue_fn->enq_multi((queue_t)queue, buf_hdr, num);
```

Comment:
I'd prefer some type safety here, so no direct casts, but an inline function that does that.